auslib/web/common/releases_spec.yml

```diff
@@ -0,0 +1,129 @@
+          -If no arguments are provided, it returns information about all of the Releases;
+          -If "product" is passed, only Releases with the given product name will be returned;
+          -If "name_prefix" is passed, only Releases whose name starts with the given prefix will be returned;
+          -If "names_only" is set to true, the response changes format and provides a list of all of the Release names.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/productQueryParam'
+        - name: name_prefix
+          description: If "name_prefix" is passed, only Releases whose name starts with the given prefix will be returned.
+          in: query
+          required: false
+          type: string
+        - name: names_only
+          description: If "names_only" is set to true, the response changes format and provides a list of all of the Release names.
+          in: query
+          required: false
+          type: string
+      responses:
+        200:
+          description: A releases list or a list of releases names.
+          schema:
+            $ref: '#/definitions/ReleasesWrapper'
+
+  /api/v1/releases/{release}:
+    get:
+      operationId: 'auslib.web.common.releases.get_release'
+      description: Returns the "data" portion of the named Release, which is a JSON Object.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/releaseParam'
+        - name: pretty
+          description: If "pretty" is present in the query string and set to true, it will be pretty formatted.
+          in: query
+          required: false
+          type: integer
+      responses:
+        200:
+          description: Release Blob
+
+  /api/v1/releases/{release}/revisions:
+    get:
+      operationId: 'auslib.web.common.releases.get_release_history'
+      description: Returns previous versions of the named Release in a JSON Object.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/releaseParam'
+        - name: page
+          in: query
+          required: false
+          type: integer
+          format: int32
+        - name: limit
+          in: query
+          required: false
+          type: integer
+          format: int32
+      responses:
+        200:
+          description: Release History.
+        400:
+          description: Bad input.
+
+  /api/v1/releases/{release}/builds/{platform}/{locale}:
+    get:
+      operationId: 'auslib.web.common.releases.get_release_single_locale'
+      description: Returns the platform+locale specific data of the named Release, which is a JSON Object.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/releaseParam'
+        - name: platform
+          description: Platform
+          in: path
+          required: true
+          type: string
+        - $ref: '#/parameters/localeParam'
```

Comment:
Maybe I did not understand what you mean, but Yes, this endpoint returns exactly this. 
As in: https://github.com/mozilla/balrog/blob/master/auslib/web/admin/views/releases.py#L191